helper-cli/src/main/kotlin/commands/curationshelper/CurationHelperCommand.kt

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright (C) 2021 Bosch.IO GmbH
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+package org.ossreviewtoolkit.helper.commands.curationshelper
+
+import com.github.ajalt.clikt.core.CliktCommand
+import com.github.ajalt.clikt.core.subcommands
+
+internal class CurationHelperCommand : CliktCommand() {
```

Comment:
I'd prefer doing the "organization" thing later when there is more knowledge. Now there is just one command so it does not make too much sense as-is, because the context of commands which will be added in the future is unknown. Maybe other options for organizing that pop up by then.

Reply:
As I added a second command in a later commit of this PR. Is your change request still valid?

Reply:
no, I wasn't aware when writing the comment.